galley/pkg/crd/validation/validation.go

```diff
@@ -146,6 +146,7 @@ func RunValidation(ready chan<- struct{}, stopCh chan struct{}, vc *WebhookParam
 			if readinessProbeController != nil {
 				validationReadinessProbe.SetAvailable(errors.New("stopped"))
 			}
+			// nolint: staticcheck
```

Comment:
What is this needed for?

Reply:
should not be needed now that https://github.com/istio/istio/commit/3cb2c39d24a449896a778b5739512c8c122ede5b merged